explanations/best-practices/code-reviews.rst

```diff
@@ -0,0 +1,41 @@
+============
+Code reviews
+============
+
+Writing code is a craft. There is no absolute right or wrong way of expressing thoughts into code. It's mainly contextual:
+
+- What are the current needs?
+- What are the current constraints?
+- What is our current understanding of the problem?
+
+Reviewing code is also contextual. Here below are some guidelines. They must not be applied in every circumstance. Instead, they define a baseline in terms of expectations on the team.
+
+Requesting a review
+===================
+
+- **Load:** requests should be well balanced among team members so we don't overload a single individual. GitHub's CODEOWNERS files can help with this.
```

Comment:
I would argue CODEOWNERS files do the opposite?
I think we also talked about requesting review from the group as a general rule, except in cases where review from a specific person as subject matter expert is desired?

Reply:
Depends if the CODEOWNERS lists a single person or a group. In the latter case it can force people to use the group.. but good call out. It could also make things worse.

Reply:
Okay. Should we mention anything about CODEOWNERS, then?

Reply:
I would leave it out.

Reply:
Okay. Removed.